create_aio_app/template/hooks/post_gen_project.py

```diff
@@ -8,20 +8,26 @@
 )
 
 
+def remove_database_redis_files() -> None:
```

Comment:
U use only one wait script, so remove only one file. And i think` remove_wait_script` is better name for this function